<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="inputField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>691</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 25pt &quot;Times&quot;;
	border: 1px solid gray;
	border-radius: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_all_clear">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  20px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>240</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(138, 180, 248);
    font-size:  16px;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_float">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>290</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(60, 64, 67);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_unar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="resultField">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 14pt &quot;Times&quot;;
	color: rgb(110, 109, 111);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(95, 99, 104);
    font-size:  16px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_value">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>90</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(138, 180, 248);
    font-size:  16px;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: qlinearegradient(x1:0, y1:0, x2:0, y2:1,
								 stop: 0 #dadbde, stop: 1 #f67fa);
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
